web/public/js/services/event-utils.js

```diff
@@ -2,15 +2,33 @@
 
 angular.module('cpZenPlatform').factory('eventUtils', ['$translate', function($translate){
   var eventUtils = {};
-  eventUtils.isEventInPast = function(dateObj){
+  eventUtils.isEventInPast = function (dateObj) {
     var now = moment.utc();
     var eventUtcOffset = moment(dateObj.startTime).utcOffset();
     var start = moment.utc(dateObj.startTime).subtract(eventUtcOffset, 'minutes');
 
     return now.isAfter(start);
   }
 
-  eventUtils.getFormattedDates = function(event) {
+  eventUtils.getNextDates = function (dates, formattedDates) {
+    var nextDateIndex = void 0;
+    var eventIndex = 0;
+    while (eventIndex < dates.length && _.isUndefined(nextDateIndex)) {
+      var date = dates[eventIndex];
+      if (!eventUtils.isEventInPast(date)) {
+        nextDateIndex = eventIndex;
+      }
+      eventIndex +=1;
+    }
+
+    if (nextDateIndex >= 0 && formattedDates) {
+      return formattedDates.slice(nextDateIndex);
+    }
+    // Congrats, you called this function upon an past event
+    return false;
```

Comment:
I'd rather `return [];` here since we otherwise return an array of dates. Makes more sense to go `_.isEmpty(nextDates)` than `nextDates === false` to me.